soc/realtek/ec/rts5912/reg/reg_adc.h

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (c) 2025 Realtek, SIBG-SD7
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_SOC_REALTEK_RTS5912_REG_ADC_H
+#define ZEPHYR_SOC_REALTEK_RTS5912_REG_ADC_H
+
+typedef struct {
+	volatile uint32_t CTRL;
+	volatile uint32_t CHCTRL;
+	volatile uint32_t STS;
+	volatile uint32_t CHDATA[12];
+	volatile uint32_t COEFFA;
+	volatile uint32_t COEFFB;
+} ADC_Type;
+
+/* CTRL */
+#define ADC_CTRL_EN_Pos         (0U)
+#define ADC_CTRL_EN_Msk         BIT(ADC_CTRL_EN_Pos)
```

Comment:
The `_Msk` suffix implies that this field is multiple bits.  You can simplify by dropping the ` _Msk` suffix.  The `_Pos` helper macro makes this harder to read so I recommend deleting it.  Do this for all the macros define for the ADC.

```suggestion
#define ADC_CTRL_EN         BIT(0)
```